lib/generator.js

```diff
@@ -835,6 +842,42 @@ class Generator {
       if (!isDirEmpty) throw new Error(`"${this.targetDir}" is not an empty directory. You might override your work. To skip this rule, please make your code a git repository or use the --force-write flag (not recommended).`);
     }
   }
+
+  /**
+   * Creates a custom resolver that maps mapBaseUrlToFolder.url to mapBaseUrlToFolder.folder
+   *
+   * @private
+   * @param  {string} fileName Name of the file to check against a list of glob patterns.
+   * @return {{read(*, *, *): Promise<unknown>, canRead(*): boolean, order: number}}
+   */
+  getMapBaseUrlToFolderResolver(mapBaseUrlToFolder) {
```

Comment:
This function is inside class, so you can use `this` for `mapBaseUrlToFolder` argument :)

```js
getMapBaseUrlToFolderResolver() {
...
const baseUrl = this.mapBaseUrlToFolder.url;
const baseDir = this.mapBaseUrlToFolder.folder;
```

Reply:
good eyes :) Thanks!